extensions/resource-deployment/src/ui/validation/validations.ts

```diff
@@ -59,27 +62,40 @@ export abstract class Validation {
 		return this._valueGetter();
 	}
 
-	protected getVariableValue(variable: string): Promise<ValidationValueType> {
-		return this._variableValueGetter!(variable);
+	protected getTargetValue(variable: string): Promise<ValidationValueType> {
+		return this._targetValueGetter!(variable);
 	}
 
-	constructor(validation: ValidationInfo, protected _valueGetter: ValueGetter, protected _variableValueGetter?: VariableValueGetter) {
+	protected async onValidation(isValid: boolean): Promise<void> {
+		await this._onValidation(isValid);
```

Comment:
_onValidation is already protected - why not just use that directly instead of creating this?

Reply:
Ping